TriggerRepoBuild/Program.cs

```diff
@@ -42,7 +42,7 @@ private static void Process(Options options)
                     RequestRepository repoObject = JsonConvert.DeserializeObject<RequestRepository>(repoResponse.Content.ReadAsStringAsync().Result);
 
                     Console.WriteLine($"Processing: {repoObject.owner.username}/{repoObject.name}");
-                    if (IsStuck(repoObject.owner.username, repoObject.name, options.RenderUrl))
+                    if (IsStuck(repoObject.owner.username, repoObject.name, options.RenderUrl) || options.ForceRender)
                     {
                         Console.WriteLine("Repo is stuck");

```

Comment:
This log entry is now incorrect, because the repo might be stuck or it might be forced.

```suggestion
                        Console.WriteLine("Triggering render");
```